src/routes/participant/availability/+page.svelte

```diff
@@ -11,8 +12,14 @@
 		StructuredListRow
 	} from 'carbon-components-svelte'
 	import { CheckmarkOutline, Launch, MisuseOutline } from 'carbon-icons-svelte'
+	import { getTimeString } from '$lib/util/timeString'
 
-	export let data: { rehearsals: rehearsal[] }
+	export let data: { rehearsals: rehearsal[]; availability: availability[] }
+
+	let rehearsalAvailability = Object.assign(
+		{},
+		...data.availability.map((x) => ({ [x.rehearsal.id]: x.available }))
+	)
```

Comment:
Die x.available is een beetje een misuse van waar het voor bedoeld was. Het idee was dat je per repititie een paar records met available.true had voor wanneer je wel kan, en available.false voor wanneer je niet kan. Als er dan iets niet was weet je dat het niet ingevuld is. Nu is het denk ik onmogelijk om te onderscheiden tussen niet ingevuld en de hele avond unavailable right?

Reply:
Jawel, want als je niet ingevuld hebt heb je geen availability doc, en dus geen availability.id daar :p

Reply:
right, zag later pas dat je de available op false zet als iemand de hele tijd niet available is. Which is good enough voor nu I guess